.eslintrc.json

```diff
@@ -19,6 +19,7 @@
     "no-console": 0,
     "newline-per-chained-call": 0,
     "prefer-const": 0,
+    "linebreak-style": 0, // due to thousands of "Expected linebreaks to be 'LF' but found 'CRLF'  linebreak-style" errors on windows
```

Comment:
This is no longer valid JSON.

Reply:
Was it changed again? [See this](https://eslint.org/docs/user-guide/configuring#comments-in-configuration-files).

Reply:
Oh never mind, ESLint must remove the comments before parsing the JSON. Vanilla JSON does not support comments:

```
> JSON.parse('{"foo":"bar"} // comment')
SyntaxError: Unexpected token / in JSON at position 14

```